specs/commandline/sign.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "notation sign"
+description: "The notation sign command description and usage"
+keywords: "notation, sign"
```

Comment:
I think this header is not necessary, we can remove this for now.

Reply:
Agree. We can remove it untill there is a valid usage.